content/en/infrastructure/livecontainers.md

```diff
@@ -13,73 +13,100 @@ further_reading:
 
 {{< img src="infrastructure/livecontainers/livecontainersoverview.png" alt="Live containers with summaries"  >}}
 
-### Kubernetes Resources
-
-[Kubernetes Resources for Live Containers][1] is currently in private beta. Fill out [this form][5] to request access.
-
-If you're using Kubernetes, enable Kubernetes Resources for Live Containers to gain multi-dimensional visibility into all Kubernetes workloads across your clusters. Inspired by the `kubectl` tool, this feature gives you complete coverage of your Kubernetes infrastructure in a continuously updated table with curated resource metrics, faceted search, per-workload detailed view, and visualized maps.
-
-## Installation
-
-Follow the [Docker][6] or [Kubernetes][7] Agent installation instructions. Enable the Process Agent to populate your Live Containers view. Container metrics are available without additional configuration after installation.
-
-**Kubernetes Resources for Live Containers requires installation of**:
-
-* [Datadog Agent][8] version 7.21.1 (or above)
-* [Datadog Cluster Agent][9] 1.9.0 (or above)
+## Configuration
 
 ### Kubernetes Resources
 
-To enable Kubernetes Resources for Live Containers, follow the [Helm instructions][10] and add the following changes to your `values.yaml` file:
-
-{{< code-block lang="yaml" filename="values.yaml" >}}
-datadog:
-  ...
-  processAgent:
-    enabled: true
-  ...
-  orchestratorExplorer:
-    enabled: true
-...
-clusterAgent:
-  enabled: true
-  image:
-    repository: datadog/cluster-agent
-    tag: latest
-    pullPolicy: Always
-...
-agents:
-  image:
-    repository: datadog/agent
-    tag: latest
-    pullPolicy: Always
-...
-{{< /code-block >}}
-
-In cases where the Agent is not able to automatically detect the Kubernetes cluster name, you must set it in `values.yaml`:
-
-{{< code-block lang="yaml" filename="values.yaml" >}}
-datadog:
-   ...
-   clusterName: <PLACEHOLDER>
-   ...
-{{< /code-block >}}
-
-**Note**: The cluster name must be 80-characters or less.
-
-On Google's GKE, AWS EKS, and Azure AKS, this is unnecessary, unless the Agent and the cluster Agent don't have access to the cloud metadata APIs, or the cluster name is longer than 80 characters.
+The Agent and [Cluster Agent][5] can be configured to retrieve Kubernetes resources for [Live Containers][6]. This feature allows you to monitor the state of pods, deployments and other Kubernetes concepts in a specific namespace or availability zone, view resource specifications for failed pods within a deployment, correlate node activity with related logs, and more.
+
+Kubernetes resources for Live Containers requires [Agent version >= 7.21.1][7] prior to the configurations below.
+
+{{< tabs >}}
+{{% tab "Helm" %}}
+
+If you are using the official [Datadog Helm Chart][1]:
+
+- Use chart version 2.4.5 or above
```

Comment:
we should also advise users to ensure that if they're hardcoding agent and cluster-agent versions they need to update them to the minimum versions required or above. Otherwise some of them will set the option but run old agents and it won't work.

Reply:
Ah, good catch: do they need to update the version in the agent config, or within the helm chart?

Reply:
in the helm chart's values.yaml